src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/SolutionRepository.cs

```diff
@@ -29,7 +29,7 @@ public sealed class SolutionRepository : ISolutionRepository
                             s.ClientApplication,
                             s.Hosting,
                             s.LastUpdated AS SolutionDetailLastUpdated,
-                            f.IsFoundation
+                            (SELECT MAX(CONVERT(int,IsFoundation)) FROM dbo.FrameworkSolutions WHERE SolutionId = @id GROUP BY SolutionId) AS IsFoundation
```

Comment:
I think we should use `CAST(f.IsFoundation AS int)` instead of `CONVERT` – just because I think it's better to prefer ANSI standard SQL where possible.

Reply:
Added two scenarios in feature file